Describe how to import old hash onion in Lisk docs (#1841)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc

----

== Set the hash onion

Without the hash onion, a validator won't be able to receive any rewards for generating new blocks, although the blocks would still be valid in that case.
In order to ensure that no rewards are missed, it is crucial for a validator to set the hash onion before enabling block generation on the node.

> IMPORTANT: When attempting to migrate to a new or different node, it is crucial that the hash onions utilized during the configuration process correspond to the previous node.
> Use the previously stored hash onion to successfully migrate information from the previous node to a newer node.

Reviewer: ```suggestion
IMPORTANT: When attempting to migrate to a new or a different node, it is crucial that the hash onions from the previous node are used.
Use the previously stored hash onion seed values such as the `address`, `seed`, `count`, and `distance` to successfully migrate information from the previous node to a newer node.
```

> --

Reviewer: Line 387:
```
IMPORTANT: If you're trying to migrate to a new/different node, it is essential that the hash onions used in the setting process belong to the former node.
Use the previously stored hash onion to successfully migrate information from the former node to a newer node.
```

Currently, there is nothing else to do with the hash onion, after it is created.
There are some situations though, where it is needed to get the seed hash onion back, for example, when xref:{url_run_block_generation_move}[moving from one node to another].
In that case, retrieve the hash onions from the node as described below, and store them in a secure location.

> Later on, these values can be used to Import hash onions to a newer node.

Reviewer: ```suggestion
Later on, these values can be used to import hash onions to a newer node.
```

> To get the hash onion seed of a validator, request the `random_getHashOnionSeeds` endpoint.

Reviewer: ```suggestion
To get the hash onion seed, request the `random_getHashOnionSeeds` endpoint.
```
